dynamo/plot/scVectorField.py

```diff
@@ -895,6 +1114,7 @@ def streamline_plot(
     save_show_or_return: str = "show",
     save_kwargs: dict = {},
     s_kwargs_dict: dict = {},
+    projection="2d",
```

Comment:
please don't change the streamline plot. you cannot make 3D streamline plot in matplotlib. please revert back

Reply:
This line will be removed.

Reply:
I was saying revert all code changes to streamline plot function because you cannot make 3d streamline plots